### DISCUSSION TITLE
Setting `laststatus` in `lua/config/options.lua` not taken into account

### DISCUSSION THREAD

**Discussion by jean-francois-labbe** [LazyVim/LazyVim#3486] [Q&A] [answered]
`~/.config/nvim/lua/config/options.lua
`
```
-- Options are automatically loaded before lazy.nvim startup
-- Default options that are always set: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/options.lua
-- Add any additional options here
--
vim.opt.laststatus = 2 -- one statusline per buffer
```

When I start vim I have `laststatus = 3` (default value in lazyvim config)

The `options.lua` file seems loaded but laststatus setting is not respected.

I don't understand why, any clue ?

**Comment by dpetka2001** [comment]
That's probably because in Lualine `globalstatus = true` and that must be overriding your setting.

**Comment by folke** [comment, marked as answer]
Fixed in https://github.com/LazyVim/LazyVim/commit/4d8d597a3fa10c8138641ade4981a1ce81c45910

**Reply by jean-francois-labbe** [reply]
Thanks a lot for your quick fix
